meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-client.sh

```
@@ -0,0 +1,323 @@
+            shift 2
+            ;;
+        --port)
+            API_PORT="$2"
+            BASE_URL="http://${API_HOST}:${API_PORT}"
+            shift 2
+            ;;
+        --no-color)
+            RED=''
+            GREEN=''
+            YELLOW=''
+            BLUE=''
```

Shellcheck (0.11.0)

[warning] BLUE appears unused. Verify use (or export if used externally).

(SC2034)